src/providers/address/address.ts

```diff
@@ -165,10 +130,10 @@ export class AddressProvider {
     }
 
     // Regular Address: try Bitcoin and Bitcoin Cash
-    let regularAddressLivenet = Address.isValid(address, 'livenet');
-    let regularAddressTestnet = Address.isValid(address, 'testnet');
-    let regularAddressCashLivenet = AddressCash.isValid(address, 'livenet');
-    let regularAddressCashTestnet = AddressCash.isValid(address, 'testnet');
+    const regularAddressLivenet = Address.isValid(address, 'livenet');
```

Comment:
We should refactor this to:
// first part, that check BIP21:
 if (URI.isValid(str)) return true;
...
/// then... 
 if (Address.isValid(str, 'livenet')) return true;
 if (Address.isValid(str, 'testnet')) return true;
 if ... return true;
 if ... return true;

 return false;

Reply:
Done! Thanks